src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -141,6 +141,9 @@ def load_arguments(self, _):
         c.argument('identity_scope', arg_type=identity_scope_type)
         c.argument('identity_role', arg_type=identity_role_type)
 
+    with self.argument_context('policy assignment create', resource_type=ResourceType.MGMT_RESOURCE_POLICY, min_api='2019-06-01') as c:
+        c.argument('enforcement_mode', options_list=['--enforcement-mode', '-e'], help='enforcement mode.', arg_type=get_enum_type(['Default', 'DoNotEnforce']))
```

Comment:
Can you add more detailed help message for `enforcement_mode` argument?

Reply:
Done.